partitura/utils/music.py

```diff
@@ -1126,32 +1250,38 @@ def remove_silence_from_performed_part(ppart):
 
                 for t, v in zip(c_times, c_values):
                     shifted_controls.append(
-                        dict(time=max(t - start_time, 0),
-                             number=number,
-                             value=int(v),
-                             track=track,
-                             channel=channel))
+                        dict(
+                            time=max(t - start_time, 0),
+                            number=number,
+                            value=int(v),
+                            track=track,
+                            channel=channel,
+                        )
+                    )
 
     # sort controls according to time
-    shifted_controls.sort(key=lambda x: x['time'])
+    shifted_controls.sort(key=lambda x: x["time"])
 
     ppart.controls = shifted_controls
 
     # Shift notes
     for note in ppart.notes:
-        note['note_on'] = max(note['note_on'] - start_time, 0)
-        note['note_off'] = max(note['note_off'] - start_time, 0)
-        note['sound_off'] = max(note['sound_off'] - start_time, 0)
+        note["note_on"] = max(note["note_on"] - start_time, 0)
+        note["note_off"] = max(note["note_off"] - start_time, 0)
+        note["sound_off"] = max(note["sound_off"] - start_time, 0)
 
     # Shift programs
     for program in ppart.programs:
-        program['time'] = max(program['time'] - start_time, 0)
+        program["time"] = max(program["time"] - start_time, 0)
 
 
-def note_array_from_part_list(part_list, unique_id_per_part=True,
-                              include_pitch_spelling=False,
-                              include_key_signature=False,
-                              include_time_signature=False):
+def note_array_from_part_list(
```

Comment:
As in similar comments, I slightly prefer the old formatting of definitions, but I am ok with the new one if that is preferred by PEP8.